### PR TITLE
WIP: Add network config

### PR DESCRIPTION
**Pull request by TairuiWang**
add helm config to enable high bandwidth cluster network

### CODE REVIEW

File: sched/adaptdl_sched/controller.py [hunk 1]
```diff
@@ -363,6 +366,15 @@ async def _create_pod(self, job_metadata, pod_template,
...
             container.setdefault("env", [])
+            if NETWORK_INTERFACE != "default":
+                container["env"].append({
+                    "name": "NCCL_SOCKET_IFNAME",
+                    "value": NETWORK_INTERFACE
+                })
+                container["env"].append({
+                    "name": "NCCL_DEBUG",
+                    "value": "INFO"
+                })
```
Comment (aurickq) [inline sched/adaptdl_sched/controller.py:377]: Do these configs need to be applied by the controller? Can't the user submitting the job just configure their pod template accordingly?

Comment (TairuiWang) [reply sched/adaptdl_sched/controller.py:377]: ok, I will move this to NNI side.

**Comment by TairuiWang** [comment]
test on new NIC passed.

**Comment by TairuiWang** [comment]
1. unit tests needed
2. `adaptdl-cluster-config` format clarification needed

@aurickq ready for review now

**Comment by TairuiWang** [comment]
https://github.com/petuum/adaptdl/pull/62
this is the latest one

**Pull request closed without merging**